src/main/java/seedu/exercise/MainApp.java

```diff
@@ -152,7 +152,7 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
             if (exerciseBookOptional.isEmpty()) {
                 logger.info("Data file not found. Will be starting with a sample exercise database book");
             }
-            exerciseData = exerciseBookOptional.orElseGet(DefaultExerciseDatabaseUtil::getExerciseDatabaseBook);
+            exerciseData = exerciseBookOptional.orElseGet(SampleDataUtil::getSampleExerciseBook);
```

Comment:
I have seen this method change about 3 times in the recent few PRs. 

Place an if else clause here


    if (path.equals(storage.getExerciseBookFilePath()) //use sample data util exercise book

    else //use default exercise database util database book

Reply:
Rectified.